README.md

```diff
@@ -125,6 +133,13 @@ To get started:
    poetry install
    ```
 
+   If you want to develop PDF functionality or run the PDF-related tests, you will need
+   to install additional dependencies:
+
+   ```sh
+   poetry install --extras pdf
+   ```
```

Comment:
Instead of giving a second option like this, let's just say that you have to run `poetry install --all-extras` for developers and mention that it's because of the PDF dependencies.